src/views/Shipment.vue

```diff
@@ -110,19 +126,61 @@ export default defineComponent({
     },
     async completeShipment() {
       const alert = await alertController.create({
-        cssClass: "my-custom-class",
         header: "Complete Shipment",
         message:
-          "Make sure you have entered the correct quantities for each item before proceeding.",
-        buttons: ["Cancel", "Complete"],
+          (translate("Make sure you have entered the correct quantities for each item before proceeding.")),
+        buttons: [
+          {
+            text: this.$t("Cancel"),
+            role: 'cancel',
+          }, 
+          {
+            text:this.$t('Complete'),
+            handler: () => {
+              this.updateShipment();
+            },
+          }
+        ],
       });
       return alert.present();
     },
+    async updateShipment() {
+      this.items.items.filter((item: any) => {
```

Comment:
Why there are items in items? I think we should improve

Reply:
There are items in items , since data in api is itself in that format.